src/pages/Invoice/CreateInvoice.js

```diff
@@ -1,7 +1,376 @@
+        if (localStorage.getItem("customer_data"))
+            setCustomersInfo(JSON.parse(localStorage.getItem("customer_data")));
+        if (localStorage.getItem("inventory_data"))
+            setItemInfo(JSON.parse(localStorage.getItem("inventory_data")));
+    };
+    // Function to delete items
+    const removeElement = (id) => {
+        if (window.confirm("Are you sure, you eant to delete this item?")) {
+            const items = invoiceRecipentDetails.items.filter(
+                (item) => item.id !== id
+            );
+            setInvoiceRecipentDetails({ ...invoiceRecipentDetails, items });
+        }
+    };
+    const SaveInvoice = (e) => {
+        e.preventDefault();
+        // Adding to local storage
+        if (localStorage.getItem("invoice_data") == null) {
+            localStorage.setItem("invoice_data", "[]");
+        }
+        const invoiceData = JSON.parse(localStorage.getItem("invoice_data"));
+        invoiceData.push(invoiceRecipentDetails);
+        localStorage.setItem("invoice_data", JSON.stringify(invoiceData));
+        triggerNotification("Invoice created successfully", {
+            type: "success",
+        });
+        history.push("/invoice");
+    };
+    return (
+        <Fragment>
+            <Navbar opened="invoice" />
+            <div className="page-content p-5 bg-primary">
+                <form onSubmit={(e) => SaveInvoice(e)}>
+                    <div className="page-heading-wrapper mb-5 p-5">
+                        <span className="title">
+                            {" "}
+                            <FormattedMessage id="title.invoice"></FormattedMessage>{" "}
+                        </span>
+                        <button className="btn" type="submit">
+                            <i className="fa fa-save"></i> &nbsp;{" "}
+                            <FormattedMessage id="invoice.save.button"></FormattedMessage>
+                        </button>
+                    </div>
+                    <div className="d-flex py-5 flex-grow align-items-start">
+                        <div className="card-bordered p-3 mx-5">
+                            <h4 className="billto text-muted m-0 mb-3">
+                                <FormattedMessage id="invoice.bill.to"></FormattedMessage>{" "}
+                            </h4>
+                            <div className="d-flex justify-content-between">
+                                {customersInfo.length > 0 ? (
+                                    <Fragment>
+                                        {invoiceRecipentDetails.name !== "" ? (
+                                            <Fragment>
+                                                <div className="billing_details pr-3">
+                                                    <div>
+                                                        {invoiceRecipentDetails.name ||
+                                                            customersInfo[0]
```

Comment:
store selected customer detail in `invoiceRecipentDetails` object 

`customersInfo` will be removed once app state & api introduce

Reply:
it should invoiceRecipentDetails.customer.name

Reply:
I'll fix this